.gitignore

```diff
@@ -50,6 +50,7 @@ session_state_gen_test.go
 __pycache__/
 tyk.test
 tyk-gateway.pid
+*.go-e
```

Comment:
what is this gitignore for?

Reply:
some IDE or mac specific linter artefact, tried sourcing the extension via sourcegraph, didn't find it; files added into git by mistake then removed during review